crates/fuel-core/src/state/historical_rocksdb.rs

```diff
@@ -300,12 +355,95 @@ where
 
         Ok(())
     }
+
+    /// Migrates a ModificationHistory key-value pair from V1 to V2.
+    /// The migration fails if other
```

Comment:
Changed: 87ace1b3bf. Although I'll wait also for @xgreenx to check if my comment is correct, because from the code it does not look like we use the `ConflictPolicy::Fail` flag when writing directly to rocksDB.